plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx

```diff
@@ -22,24 +27,29 @@ type PaymentExtensionSuggestionListItemProps = {
 	installingPlugin: string | null;
 	setupPlugin: ( id: string, slug: string ) => void;
 	pluginInstalled: boolean;
+	acceptIncentive: ( id: string ) => void;
 };
 
 export const PaymentExtensionSuggestionListItem = ( {
 	extension,
 	installingPlugin,
 	setupPlugin,
 	pluginInstalled,
+	acceptIncentive,
 	...props
 }: PaymentExtensionSuggestionListItemProps ) => {
-	const hasIncentive = !! extension._incentive;
-	const shouldHighlightIncentive =
-		hasIncentive && ! extension._incentive?.promo_id.includes( '-action-' );
-
 	return (
 		<div
 			id={ extension.id }
 			className={ `transitions-disabled woocommerce-list__item woocommerce-list__item-enter-done ${
-				shouldHighlightIncentive ? `has-incentive` : ''
+				hasIncentive( extension ) &&
```

Comment:
I liked how you had earlier wrapped this conditional in `shouldHighlightIncentive`.

Reply:
I moved it back to how it used to be. :)